Formula/kubeval.rb

```diff
@@ -0,0 +1,32 @@
+class Kubeval < Formula
+  desc "Validate Kubernetes configuration files, supports multiple Kubernetes versions"
+  homepage "https://www.kubeval.com/"
+  url "https://github.com/instrumenta/kubeval/archive/v0.16.1.tar.gz"
+  sha256 "a6a8a34d58af25ff111ad68c2781128a05960a04ee27ca8ff39dc20f369f854e"
+  license "Apache-2.0"
+  head "https://github.com/instrumenta/kubeval.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.version=#{version}
+      -X main.commit=#{tap.user.downcase}
```

Comment:
Is this correct?

Reply:
yeah, it was kind of intentional. But I can switch `tag-revision` checkout.

Reply:
Yes, tag-revision seems to be more appropriate in this case.

Reply:
cool, let me switch to `tag-revision` checkout.